mobile-app/app/components/OceanInterface/OceanInterface.tsx

```diff
@@ -138,10 +146,18 @@ export function OceanInterface (): JSX.Element | null {
         })
         .catch((e: Error) => {
           const errMsg = `${e.message}. Txid: ${transaction.tx.txId}`
+          const err = errorMessageMapping(errMsg)
+          notificationObj = {
+            title: translate('screens/OceanInterface', 'Transaction Error'),
+            body: translate('screens/OceanInterface', err.message)
+          }
           setError(errMsg)
           Logging.error(e)
         })
         .finally(() => {
+          if ((notificationObj.title !== '') && (notificationObj.body !== '')) {
+            void WalletNotifications.send(notificationObj)
+          }
```

Comment:
no point separating the cause and effect?

You split them up into the assignment of object, setting of object, and dispatching which is redundant IMO

You could just dispatch them via `void WalletNotifications.send({...})` for more simple implementation and lower cognitive complexity

Reply:
will do the changes as suggested